pkg/apis/rbac/validation/validation_test.go

```diff
@@ -558,3 +559,10 @@ func TestValidateRoleNoResources(t *testing.T) {
 		field:   "rules[0].resources",
 	}.test(t)
 }
+
+func nonResourceURLCovers(ownerPath, subPath string) bool {
```

Comment:
in fact we have a copy in the registry. Will move the test and remove this instance.